pkg/ext-proc/backend/vllm/metrics.go

```diff
@@ -90,35 +95,55 @@ func promToPodMetrics(metricFamilies map[string]*dto.MetricFamily, existing *bac
 	}
 	*/
```

Comment:
@liu-cong  do we have an algorithm proposal to use this metric? if not, can we cleanup this code?